.github/workflows/terraform-plan-env.yml

```diff
@@ -23,6 +46,7 @@ jobs:
       TERRAFORM_PRE_RUN: |
           apt-get update
           apt-get install -y zip
+      TF_VAR_postgrest: ${{ needs.fetch-versions.outputs.postgrest }}
```

Comment:
Noice! I didn't know you could refer to outputs this way. You should also be doing this in `terraform-apply-env.yml`.